sdks/python/apache_beam/examples/snippets/transforms/aggregation/max.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
```

Comment:
Does each of these need their own file?

Reply:
Yes, these are for the [transform catalog](https://beam.apache.org/documentation/transforms/python/overview/), so even if it is very similar content, it is useful to see examples for every single transform and how to use them in different circumstances. We already have the [element-wise transforms](https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/snippets/transforms/elementwise) completed with their docs, code samples and interactive notebooks. ([example](https://beam.apache.org/documentation/transforms/python/elementwise/map/))

We are organizing them as one file per transform, for consistency with how the docs are organized. I was going to add reviewers after tests passed since Ahmet is OOO